BMS.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+#include "BMS.h"
+
+const char* getStatusMessage(BatteryStatus status) {
+  switch (status) {
+    case TEMPERATURE_LOW: return "Temperature too low!";
+    case TEMPERATURE_HIGH: return "Temperature too high!";

```

Comment:
complexity moved from isBatteryOk to getStatusMessage ?

Reply:
I am not sure from where this other batteryIsOk(whose CCN is 6) is coming in Build and Run, In order to check I have simplified code as well. Can you help me here?